<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnON">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOFF">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTe">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Temp : 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LED</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHu">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Humid : 0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTH">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>T/H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDIST">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Dist</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDis">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance : 0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
